examples/quickstart/notebooks/quickstart.ipynb

```diff
@@ -702,33 +716,55 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "!zenml model-deployer models list"
+    "# Register the MLflow model deployer\n",
+    "!zenml model-deployer register mlflow --flavor=mlflow\n",
+    "\n",
+    "# Register a new stack with the new stack components\n",
+    "!zenml stack update quickstart -d mlflow"
    ]
   },
   {
    "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Inspecting the outcomes"
+    "Once again there are some dependencies in terms of how the step needs to run, so\n",
+    "we specify those upfront. For example, the prediction service needs to be loaded\n",
+    "before we try to make predictions with it."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "@pipeline\n",
+    "def deploy_and_predict() -> None:\n",
+    "    \"\"\"Deploy the best model and run some predictions.\"\"\"\n",
+    "    prediction_service_loader.after(model_deployer)\n",
+    "    predictor.after(prediction_service_loader)\n",
```

Comment:
Same as for the src/ definition, this `after` call is redundant since it's already required by the pipeline DAG
```suggestion
```